start.sh

```diff
@@ -6,7 +6,7 @@ if [ ! -f norduni/compose.yml ]; then
 fi
 
 # add ni.localenv.loc to hosts file
-(printf '127.0.0.1\tsri.localenv.loc\n127.0.0.1\tni.localenv.loc\n';
+(printf '127.0.0.1\tsri.localenv.loc\n';
```

Comment:
Putting stuff on 127.0.0.1 is not great when you have more than one developer repo that you want to use. It is better to set an ip for the containers you want to add to the hosts file in the compose file and then use that address.